rskj-core/src/main/java/co/rsk/net/sync/PeersInformation.java

```diff
@@ -113,13 +118,23 @@ public SyncPeerStatus getOrRegisterPeer(Peer peer) {
         return this.registerPeer(peer);
     }
 
+    @Override
+    public SyncPeerStatus getOrRegisterSnapPeer(Peer peer) {
+
+        if (syncConfiguration.getNodeIdToSnapshotTrustedPeerMap().containsKey(peer.getPeerNodeID().toString())) {
```

Comment:
not sure we need this check. Boot peers are just a fallback mechanism, other peers may also provide snap service, especially after introducing snap discovery in future. I'd rather keep it as it was before

Reply:
solved, removed this method.